setup.py

```diff
@@ -44,7 +44,7 @@ def read_version():
     author_email='aws-sam-developers@amazon.com',
     url='https://github.com/awslabs/aws-lambda-builders',
     license='Apache License 2.0',
-    packages=find_packages(exclude=('tests', 'docs')),
+    packages=find_packages(exclude=['tests.*', 'tests']),
```

Comment:
why remove the `docs` folder from here?

Reply:
`docs` folder is not a module, so this didn't do anything anyways.